inttest/common/util.go

```diff
@@ -170,6 +170,18 @@ func WaitForDeployment(ctx context.Context, kc *kubernetes.Clientset, name, name
 		})
 }
 
+// WaitForStatefulSet waits for the StatefulSet with the given name to become
+// available as long as the given context isn't canceled.
```

Comment:
I think the docs could be clearer here. This function waits until all replicas are ready, not for the "available" condition. (That condition probably doesn't exist for StatefulSets?)

Reply:
That's a fair point, since I'm here I'll make the same change for the function WaitForDaemonset